lib/desired-caps.js

```diff
@@ -110,6 +110,17 @@ validator.validators.deprecated = function (value, options, key) {
   }
   return null;
 };
+validator.validators.inclusionCaseInsensitive = function (value, options) {
+  if (typeof value !== 'string') {
+    return `unrecognised`;
```

Comment:
for consistency with above used strings i added that quotes. Output will be like ` platformName unrecognised.`